lib/src/androidTest/java/at/bitfire/vcard4android/contactrow/EventBuilderTest.kt

```diff
@@ -140,6 +142,17 @@ class EventBuilderTest {
         }
     }
 
+    @Test
+    fun testType_BirthdayWithTimezone() {
+        EventBuilder(Uri.EMPTY, null, Contact().apply {
+            birthDay = Birthday(
+                ZonedDateTime.of(1984, /* zero-based */ 7, 20, 0, 0, 0, 0, ZoneOffset.UTC)
+            )
+        }, false).build().also { result ->
+            assertEquals(CommonDataKinds.Event.TYPE_BIRTHDAY, result[0].values[CommonDataKinds.Event.TYPE])
```

Comment:
Here we don't want to check `Event.TYPE`, but whether the `Event.START_DATE` is set as expected.